homeassistant/components/cast/media_player.py

```diff
@@ -136,6 +143,29 @@ def _discover_chromecast(hass: HomeAssistantType, info: ChromecastInfo):
     dispatcher_send(hass, SIGNAL_CAST_DISCOVERED, info)
 
 
+def _remove_chromecast(hass: HomeAssistantType, info: ChromecastInfo):
+    # Removed chromecast
+    _LOGGER.debug("Removed chromecast %s", info)
+
+    dispatcher_send(hass, SIGNAL_CAST_REMOVED, info)
+
+
+class ChromeCastZeroconf:
```

Comment:
This is just a helper to store the zeroconf object instantiated by the browser to allow it to be reused.